### PR TITLE
Added logs to docker update script

### PR DESCRIPTION
**Pull request by pablomendezroyo**

### CODE REVIEW

File: packages/dappmanager/hostScripts/docker_compose_update.sh [hunk 1]
```diff
@@ -6,46 +6,60 @@ shopt -s nocasematch
...
+create_log_file
 
 function get_docker_compose_version() {
+  echo -e "\e[32mRetrieving compose version\e[0m" >> $LOG_FILE
   # Check if docker compose exists
   if type docker-compose >/dev/null 2>&1; then
     DOCKER_COMPOSE_VERSION=$(docker-compose version --short)
   else 
-    echo "docker compose does not exist or not recognized"
+    echo "docker compose does not exist or not recognized" 2>&1 | tee -a $LOG_FILE
```
Comment (3alpha) [inline packages/dappmanager/hostScripts/docker_compose_update.sh:28]: echo will always output only to stdout

Comment (pablomendezroyo) [reply packages/dappmanager/hostScripts/docker_compose_update.sh]: good point, fixed with last commit

File: packages/dappmanager/test/integration/hostScripts.test.int.ts [hunk 2]
```diff
@@ -7,21 +7,21 @@ const hostScriptsPath = process.cwd() + "/hostScripts";
 describe("Host scripts", () => {
   it("Should fetch docker engine update requirements", async () => {
     const dockerEngineUpdateRequirements = await shell(
-      `bash ${hostScriptsPath}/docker_engine_update.sh --print-host-info`
+      `sudo bash ${hostScriptsPath}/docker_engine_update.sh --print-host-info`
```
Comment (dapplion) [inline packages/dappmanager/test/integration/hostScripts.test.int.ts:10]: Why do you need sudo now?

Comment (pablomendezroyo) [reply packages/dappmanager/test/integration/hostScripts.test.int.ts:10]: because it creates a log file at a reserved path: /usr/src/dappnode/logs

Comment (dapplion) [reply packages/dappmanager/test/integration/hostScripts.test.int.ts:11]: Please find an alternative way to do that. Using sudo just for logs is not good

Comment (pablomendezroyo) [reply packages/dappmanager/test/integration/hostScripts.test.int.ts:11]: https://github.com/dappnode/DNP_DAPPMANAGER/pull/748#discussion_r625614512

**Pull request merged**